rllib/agents/sac/sac_torch_policy.py

```diff
@@ -0,0 +1,344 @@
+from gym.spaces import Discrete
+import logging
+
+import ray
+import ray.experimental.tf_utils
+from ray.rllib.agents.a3c.a3c_torch_policy import apply_grad_clipping
+from ray.rllib.agents.sac.sac_tf_policy import build_sac_model, \
+    postprocess_trajectory
+from ray.rllib.agents.dqn.dqn_tf_policy import PRIO_WEIGHTS
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.policy.torch_policy_template import build_torch_policy
+from ray.rllib.models.torch.torch_action_dist import (
+    TorchCategorical, TorchSquashedGaussian, TorchDiagGaussian, TorchBeta)
+from ray.rllib.utils import try_import_torch
+
+torch, nn = try_import_torch()
+F = None
+if nn is not None:
+    F = nn.functional
```

Comment:
This should be unnecessary on master; functional is stubbed out.

Reply:
+1